Makefile.am

```diff
@@ -37,6 +37,7 @@ SUBDIRS += tests
 
 VENV_DIR = $(HOME)/.venv/vdsm
 VENV = $(VENV_DIR)/bin
+RPM_TOPDIR = $(PWD)/build/rpm
```

Comment:
In imageio we use a sub directory because we use the same build directory python used when running "python setup.py build", and after buidling rpms, we move them the output dir. In vdsm the build directory is used only for rpms, so we can simplify.